### PR TITLE
Spearhead: change colors for dark mode mobile menu

### LINKED ISSUE #2668: Spearhead: Dark mode mobile menu
@kjellr Should we have the menu on mobile be dark on dark mode?

Screenshot 2020-10-22 at 12 22 20

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

Changed the colors on darkmode to follow the new design guidelines from the related issue.

* Light mode should stay the same as before:

Screenshot 2020-10-27 at 17 12 06

* Dark mode should have the light background color we have under the sticky post and white links:

Screenshot 2020-10-27 at 17 09 07

#### Related issue(s):

Closes #2668

### CODE REVIEW

File: spearhead/variables.css [hunk 1]
```diff
@@ -108,10 +109,11 @@
...
 		--global--color-text-selection: #000000;
-		--primary-nav--color-background: var(--global--color-foreground-low-contrast);
-		--button--padding-vertical: 24px;
-		--button--padding-horizontal: 32px;
 		--sticky-posts--color-background: var(--global--color-background);
 		--sticky-posts--alt-color-background: #2d3139;
+		--primary-nav--color-background: var(--sticky-posts--alt-color-background);
+		--primary-nav--dropdown-color-link: var(--global--color-foreground);
+		--button--padding-vertical: 24px;
+		--button--padding-horizontal: 32px;
```
Comment (scruffian) [inline spearhead/variables.css:117]: Why are you only changing the button padding in dark mode?

**Comment by kjellr** [comment]
Aside from that question from @scruffian above, this looks good to me! Colors change as expected.

File: spearhead/variables.css (hunk 1 above)
Comment (MaggieCabrera) [reply spearhead/variables.css:117]: No, I'm  not changing the padding, I changed the variable order only. The padding values are the same.

Comment (scruffian) [reply spearhead/variables.css:117]: Oh yeah I see now

**Review by scruffian — APPROVED** [review]

**Pull request merged**